access_point/server/server.py

```diff
@@ -13,19 +17,19 @@ def install():
 
 @app.route('/config/<stt>/<tts>/<hotword>/<wake>')
 def config(stt, tts, hotword, wake):
-    subprocess.call(['sudo', 'bash' , 'server/config.sh', stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
+    subprocess.call(['sudo', 'bash' , server_folder + '/config.sh', stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
```

Comment:
Should build path with `os.path.join`, not `+` operator.

Reply:
`os.path.join` may be better if we would expect that this script must work in non-linux (i.e. Windows) environments. But that is not the case. Therefore I think its not too bad to just use '+'